gradio/routes.py

```diff
@@ -332,6 +338,15 @@ async def predict(
                         },
                         status_code=500,
                     )
+            if not app.blocks.api_open and app.blocks.queue_enabled_for_fn(
+                body.fn_index
+            ):
+                if f"Bearer {app.queue_token}" != request.headers.get("Authorization"):
```

Comment:
I think `Bearer {token}` in the `Authorization` header is the standard way of providing these tokens.